<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="info.antoniomartin.minesweeper.domain.CellTest" tests="9" skipped="0" failures="0" errors="0" timestamp="2021-10-23T18:57:13" hostname="Erytheia" time="0.017">
  <properties/>
  <testcase name="The_cell_is_open()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.005"/>
  <testcase name="I_have_a_mine()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.001"/>
  <testcase name="The_cell_is_a_flag()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.001"/>
  <testcase name="I_am_open()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.0"/>
  <testcase name="I_dont_have_a_mine()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.0"/>
  <testcase name="someone_putMe_a_flag()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.0"/>
  <testcase name="I_am_exploded()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.0"/>
  <testcase name="Quit_a_flag_from_cell()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.001"/>
  <testcase name="I_am_close()" classname="info.antoniomartin.minesweeper.domain.CellTest" time="0.001"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
